docs/tutorials/build-a-dapp/pallet.md

```diff
@@ -104,40 +115,53 @@ use frame_system::ensure_signed;
 use sp_std::vec::Vec;
 ```
 
+> Critical: the `no_std` feature is **required** for all pallets! This is because we are building a
```

Comment:
For consistency in style, I'd suggest to bold "Critical" :)

Reply:
Sure - messaged you in element. Feel free to make edits